xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -430,6 +430,9 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item,
       case LISTITEM_CHANNEL_NUMBER:
       case LISTITEM_PREMIERED:
       case VIDEOPLAYER_PARENTAL_RATING:
+      case VIDEOPLAYER_PARENTAL_RATING_CODE:
+      case VIDEOPLAYER_PARENTAL_RATING_ICON:
+      case VIDEOPLAYER_PARENTAL_RATING_SOURCE:
```

Comment:
This wrong. Please remove all four lines. First is already removed in latest master. Please rebase.
This sections is for timers only. Timers will never be played by the VideoPlayer. That's why here only LISTITEM_* makes sense.